src/styles.css

```diff
@@ -0,0 +1,170 @@
+body {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #6d6970;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: black;
+  position: fixed;
+  width: 100%;
+}
+
+#logo {
+  display: flex;
+}
+
+#logo1 {
+  height: 5rem;
+}
+
+#title {
+  color: #967843;
+}
+
+#header-nav {
+  display: flex;
+  justify-self: end;
+}
+
+#nav-list {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  gap: 300px;
+  font-size: 1.5rem;
+  cursor: pointer;
+  color: #967843;
+  margin-right: 45px;
+}
+
+footer {
+  text-align: start;
+  background-color: black;
+  margin-top: 10px;
+}
+
+.footer-section {
+  color: #967843;
+  font-size: 20px;
+  text-align: start;
+  padding-left: 10px;
+}
+
+#art-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 84px;
+  justify-content: space-evenly;
+}
+
+.article-style {
+  width: 31.5%;
+  height: 120vh;
+  margin-top: 20px;
+  background-color: black;
+  border: 3px solid #0f1429;
+  gap: 5px;
+}
+
+.hidden {
+  display: none;
+}
+
+.image-style {
+  height: 32rem;
+  width: 26.4rem;
+  align-self: center;
+  margin-left: 5%;
+  margin-top: 25px;
+  border: 5px solid #967843;
+}
+
+.bg-popup {
+  width: 60%;
+  background-color: #0f1429d1;
+  position: fixed;
+  align-self: center;
+  text-align: center;
+  margin-top: 25px;
+  height: 90vh;
+  overflow-y: auto;
+  border: 5px solid #967843;
+}
+
+.bg-popup img {
+  width: 50%;
+  height: 25rem;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  margin-top: 5px;
+}
+
+input,
+textarea {
+  margin: 0 auto;
+  padding-left: 50px;
+  padding-right: 50px;
+  border-radius: 30px;
+}
+
+button {
+  margin: 0 auto;
+  padding: 7px;
+  background-color: #967843;
+  color: #fff;
+  border-radius: 16px;
+}
+
+#closeIcon {
+  text-align: end;
+  color: #fff;
+  font-size: x-large;
+  cursor: pointer;
+  margin-left: 90%;
+}
+
+.title,
+.artist {
+  color: #967843;
+  text-align: center;
+}
+
+.like-count {
+  color: #fff;
+  margin-left: 50px;
+  margin-top: -50px;
+}
+
+figcaption {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.text-lg {
+  color: #fff;
+}
+
+.comments-counter {
+  color: #fff;
+}
+
+.like {
+  height: 40px;
+  width: 40px;
+  margin-left: -50px;
+}
+
+.comment {
```

Comment:
- [ ] [Optional] For better UI please consider using the same padding for the  name and comment fields

![Screenshot_20220520_184431](https://user-images.githubusercontent.com/25227022/169583774-4abaeb09-a939-47bf-8d21-e368d090d007.png)